man/memtier.7

```diff
@@ -127,16 +135,22 @@ with filesystem created on top of it:
 .IP
 .B MEMKIND_MEM_TIERS="KIND:FS_DAX,PATH:/tmp/,RATIO:1;POLICY:STATIC_THRESHOLD"
 .PP
-Example usage of MEMKIND_MEM_TIERING_CONFIG environment variable where 20% of allocations will go to DRAM
-and 80% of allocations will go to DAX KMEM nodes:
+Example usage of MEMKIND_MEM_THRESHOLDS environment variable. With VAL=64, on the application start all allocations
+lower than 64 bytes will go to DRAM and equal to or greater than this value will go to DAX_KMEM nodes. This threshold value
+changes during the runtime in order to maintain the ratio. MIN=1 and MAX=10000 set the lower and upper limits of this threshold
+value. Note that the DYNAMIC_THRESHOLD policy has to be set in MEMKIND_MEM_TIERS environment variable:
 .IP
-.B MEMKIND_MEM_TIERS="KIND:DRAM,RATIO:1;KIND:KMEM_DAX,RATIO:4,POLICY:STATIC_THRESHOLD"
-.B TODO Example with dynamic threshold
+.B MEMKIND_MEM_TIERS="KIND:DRAM,RATIO:1;KIND:KMEM_DAX,RATIO:4;POLICY:DYNAMIC_THRESHOLD
+.B MEMKIND_MEM_THRESHOLDS="VAL=64,MIN=5,MAX=10000"
 .SH "NOTES"
 .B libmemtier
-works for applications which does not statically link a
+works for applications which does not:
```

Comment:
Either:
works for applications which do not...
or:
works for application which does not...